.scripts/run_osx_build.sh

```diff
@@ -25,9 +25,9 @@ conda activate base
 
 echo -e "\n\nInstalling ['conda-forge-ci-setup=3'] and conda-build."
 mamba install --update-specs --quiet --yes --channel conda-forge \
-    conda-build pip boa conda-forge-ci-setup=3
+    conda-build liblief=0.11.5 pip boa conda-forge-ci-setup=3
 mamba update --update-specs --yes --quiet --channel conda-forge \
-    conda-build pip boa conda-forge-ci-setup=3
+    conda-build liblief=0.11.5 pip boa conda-forge-ci-setup=3
```

Comment:
Can you add:

```
remote_ci_setup:
- liblief=0.11.5
```
to conda-forge.yml

https://github.com/conda-forge/conda-smithy/pull/1579/files#diff-d889cd3c5e3aa36e3c852ab58fe7f8abae463e760186f7cae7b40d93646852aeR4